docs/databases/connecting.md

```diff
@@ -162,7 +160,9 @@ To forget the data that Metabase has stored from previous [database scans](#sync
 
 Go to **Admin settings** > **Databases** > your database and click **Remove this database**.
 
-![Database Manual Sync](./images/DatabaseManualSync.png)
+## Restoring the Sample Database
+
+If you've deleted the Metabase [Sample Database](https://www.metabase.com/glossary/sample_database), go to **Admin settings** > **Databases** and click **Bring the Sample Database back**.
```

Comment:
Nit:
The copy actually uses lowercase letters: "Bring the sample database back".
![image](https://user-images.githubusercontent.com/31325167/196199934-4e3de025-009d-463f-936c-dd264283d8d1.png)

Should we update the string in the code or the documentation?

Reply:
I think we should update it in the code :)